### PR TITLE
limit max capacity

### PR DESCRIPTION
**Pull request by fewieden**
I stumbled upon the issue, that on production I had way less metric points in my chart as on testing. Since the usage on production is higher this looked weird. After investigating I found a lot of errors about failed requests and therefore the limitation of 20 metrics per request https://docs.aws.amazon.com/AmazonCloudWatch/latest/APIReference/API_PutMetricData.html. Due to that I lost a lot of metric points. To prevent this for other people in the future I added a safeguard.

In addition I also changed:
- [ ] self => this
- [ ] const/let
- [ ] arrow functions
- [ ] helper functions for tests
- [ ] travis runs `lint` and `test`
- [ ] shutdown also summary interval

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,8 +4,8 @@
   "description": "A simple wrapper for simplifying using Cloudwatch metrics",
   "main": "index.js",
   "scripts": {
-    "test": "jasmine",
-    "lint": "eslint ."
+    "test": "./node_modules/.bin/jasmine",
```
[the remark this answers is not in the document]
Comment (fewieden) [reply package.json]: Reverted

I always use the local path, after we discovered last year in my company that the scripts were sometimes executed with an outdated global module. This way we ensure that it fails if the dependencies are not installed and doesn't fallback to global module.

File: spec/src/indexSpec.js [hunk 2]
```diff
@@ -16,113 +16,146 @@ describe('cloudwatch-metrics', function() {
...
+        Value: i
+      });
+    }
+
+    return metricData;
+  }
+
+  function putMetric(metric, amount) {
+    for (let i = 1; i <= amount; i++) {
+      metric.put(i, 'metricName', [{Name:'ExtraDimension',Value: 'Value'}]);
```
[the remark this answers is not in the document]
Comment (fewieden) [reply spec/src/indexSpec.js]: I found more occurrences of this spacing issue, I fixed them as well. This should probably be covered with an eslint rule.

**Comment by bradvogel** [comment]
@skeggse can this be merged?

**Comment by bradvogel** [comment]
Closing due to age. Please reopen if necessary.

**Pull request closed without merging**